pkg/scheduler/api/job_info.go

```diff
@@ -652,6 +652,36 @@ func (ji *JobInfo) CheckTaskMinAvailable() bool {
 	return true
 }
 
+// CheckTaskMinAvailableReady return ready pods meet task minavaliable.
+func (ji *JobInfo) CheckTaskMinAvailableReady() bool {
+	occupiedMap := map[TaskID]int32{}
```

Comment:
if `ji.MinAvailable < ji.TaskMinAvailableTotal`, l think we should ignore this judgement

Reply:
This has already been checked in the validJobFn function, I think there is no need to check it again here.

Reply:
I think it is necessary to add it, which means when `ji.MinAvailable <ji.TaskMinAvailableTotal`, we will skip task minAvailable judgement. Otherwise there will be some problems if task minAvailable is not set(task minAvailable == task replica)